### PR TITLE
Cli: wash docs

### PR DESCRIPTION
**Pull request by aish-where-ya**
## Feature or Problem
FEAT: wash up docs
This branch is based off of the [wash-app](https://github.com/wasmCloud/wasmcloud.com-dev/pull/173) branch. It will need refactoring before merging.

## Release Information
next

## Consumer Impact
Addition to CLI docs.

## Testing
manually tested

Built on platform(s)
- [ ] x86_64-linux
- [ ] aarch64-linux
- [ ] x86_64-darwin
- [x] aarch64-darwin
- [ ] x86_64-windows

Tested on platform(s)
- [ ] x86_64-linux
- [ ] aarch64-linux
- [ ] x86_64-darwin
- [x] aarch64-darwin
- [ ] x86_64-windows

### Unit Test(s)
NA

### Acceptance or Integration
NA

### Manual Verification
Manually verified

### CODE REVIEW

File: docs/cli/app.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+This will retrieve a list of all applications that wadm knows about. Application specifications (also referred to in our documentation and code as "models") are stored in a model store and are associated to your lattice.
+
+Usage: 
+
+```
+wash app list
+```
+
+### `get`
+The `get` command is used to retrieve the details for a specific version of a model. You can retrieve both the original text (e.g. YAML) that you submitted and the vetted and normalized model in JSON format.
```
Comment (thomastaylor312) [inline docs/cli/app.md:28]: It would probably be good to show an example of fetching it as JSON since it is mentioned here

File: docs/cli/app.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+Usage:
+
+```
+wash app history <name>
+wash app history petclinic
+```
+
+### `delete`
+The `delete` command is used to delete an application model of a specific version. You can optionally delete _all_ versions of a specificationas well.
```
Comment (thomastaylor312) [inline docs/cli/app.md]: Need a space after "specification"

File: docs/cli/app.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+Usage:
+
+```
+wash app put <SOURCE>
+wash app put petclinic.yaml
+```
+
+### `deploy`
+When a version of an application specification is deployed, wadm will start monitoring the state of your lattice. Wadm will compare the desired state of an application with its current state and issue the appropriate  commands to reconcile the state.
```
Comment (thomastaylor312) [inline docs/cli/app.md]: Remove extra space after "appropriate"

File: docs/cli/app.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+
+Usage:
+
+```
+wash app deploy <name> [version]
+wash app deploy petclinic v0.0.1
+```
+
+### `undeploy`
+Undeploying an application spec tells wadm to stop monitoring that deployment. Wadm supports _destructive_ as well as _non-destructive_ undeploys. By default, all resources originally provisioned for an application will be removed after that application is undeployed.
```
Comment (thomastaylor312) [inline docs/cli/app.md]: Currently, non-destructive undeploys are not actually functional, so we should probably remove that from here

File: docs/cli/build.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
+---
+title: "wash build"
+draft: false
+sidebar_position: 2
+description: "wash build command reference"
+---
+
+To build custom wasmCloud entities like actors, providers or interfaces, `wash build` comes in as a handy tool. One can bring in their own project containing some business logic and build these entities by simply providing the path to the project or its associated `wasmcloud.toml` file. The built artifact is signed using keys or a user may use their own keys to sign the build.
```
Comment (thomastaylor312) [inline docs/cli/build.md]: "The built artifact is signed using keys" -> "The built artifact is signed using automatically generated keys"

File: docs/cli/capture.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
+---
+title: "wash capture"
+draft: false
+sidebar_position: 3
+description: "wash capture command reference"
+--- 
+
+Wash capture helps in capturing snapshots of the state of a lattice and assists in debugging cluster invokations ucing operations like replay. This is currently an experimental feature and you would have to pass the `--experimental` flag or set the `WASH_EXPERIMENTAL` environment variable to true. This tool provides three options - enable cpature mode, disable capture mode or replay a captured file. A user must first enable capture mode by passing the `--enable` flag. They may continue to capture snapshots using the `wash capture --experimental` and these would be stored to a snapshot file in your current directory. The snapshot files can be viewed by passing the file path to the `replay` subcommand. To disable capture mode, pass the `--disable` flag.
```
Comment (thomastaylor312) [inline docs/cli/capture.md]: s/invokations/invocations

s/ucing/using

s/cpature/capture

Comment (thomastaylor312) [reply docs/cli/capture.md]: This sentence is a little unclear:

"They may continue to capture snapshots using the `wash capture --experimental` and these would be stored to a snapshot file in your current directory." 

Consider rewording to something like 

"Once capture mode is enabled, a user can capture snapshots at any time by running `wash capture --experimental`, which will download the snapshot file to the current directory"

File: docs/cli/claims.md [hunk 7]
```diff
@@ -0,0 +1,296 @@
+---
+title: "wash claims"
+draft: false
+sidebar_position: 4
+description: "wash claims command reference"
+--- 
+
+Every actor in a wasmCloud environment claims its capabilities or the features it can interact with to supplement its current behaviour. These capabilities are claimed in the form of JSON Web Tokens (JWTs). `wash claims` will assist you to generate, manage and view these JWTs for wasmCloud actors. Following are the subcommands available under `wash claims`.
```
Comment (thomastaylor312) [inline docs/cli/claims.md]: Nit: s/behaviour/behavior (I think we use the US spellings everywhere)

File: docs/cli/claims.md [hunk 8]
```diff
@@ -0,0 +1,296 @@
...
+
+- `inspect`
+- `sign`
+- `token`
+
+## `inspect`
+
+:::caution
+This subcommand will be deprecated in furture versions. Please use `wash inspect` instead.
```
Comment (thomastaylor312) [inline docs/cli/claims.md]: s/furture/future

File: docs/cli/claims.md [hunk 9]
```diff
@@ -0,0 +1,296 @@
...
+`--password` (Alias `-p`) OCI password, if omitted anonymous authentication will be used [env: WASH_REG_PASSWORD]
+
+`--insecure` Allow insecure (HTTP) registry connections
+
+`--no-cache` skip the local OCI cache
+
+
+## `sign`
+
+`wash claims sign` assists you to sign a WebAssembly component by specifying some standard capabilities available in the wasmCloud environment. A user may also specify custom capabilities and other metadata such as expiration, tags, etc.
```
Comment (thomastaylor312) [inline docs/cli/claims.md]: "assists you in signing" reads a little better

File: docs/cli/claims.md [hunk 10]
```diff
@@ -0,0 +1,296 @@
...
+
+#### Usage
+```
+wash claims token operator --name=example
+```
+
+#### Options
+`--name` (Alias `-n`) A human-readable, descriptive name for the token
+
+`--output` (Alias `-o`) Specify output format (text or json) [default: text]
```
Comment (thomastaylor312) [inline docs/cli/claims.md:217]: Super nit: A lot of these flags are global to all the subcommands. It might make it more readable and easier to maintain if we pull those out into their own section

File: docs/cli/ctl.md [hunk 11]
```diff
@@ -0,0 +1,369 @@
+---
+title: "wash ctl"
+draft: false
+sidebar_position: 6
+description: "wash ctl command reference"
+--- 
+
+This subcommand will help you to interact with the wasmCloud control interface. You may perform operations such as retrieving information about the lattice, enable provisioning or deprovisioning of entities in the lattice, creating links or updating items in the lattice. Following are the subcommands available under `wash ctl`
```
Comment (thomastaylor312) [inline docs/cli/ctl.md:8]: I would add a note up top here that the whole `wash ctl` subcommand is deprecated rather than on individual commands

File: docs/cli/ctl.md [hunk 12]
```diff
@@ -0,0 +1,369 @@
...
+- `start`
+- `stop`
+- `update`
+- `scale`
+
+## `get`
+wash get retrives information about the lattice. The following subcommands will help you retrieve different information available to query:
+
+:::caution
+This subcommand will be deprecated in furture versions. Please use `wash get` instead.
```
Comment (thomastaylor312) [inline docs/cli/ctl.md]: Looks like the `furture` fat finger was copy pasted everywhere. I'd do a quick check across all the docs here

File: docs/cli/ctl.md [hunk 13]
```diff
@@ -0,0 +1,369 @@
...
+#### Usage
+```
+wash ctl get inventory
+wash ctl get inventory <your-host-id>
+```
+
+#### Options
+These options can be used for all of the `wash ctl get` subcommands. 
+
+`--output` (Alias `-o`) Specify output format (text or json) [default: text]
```
Comment (thomastaylor312) [inline docs/cli/ctl.md:52]: Same thing with global options here

File: docs/cli/ctl.md [hunk 14]
```diff
@@ -0,0 +1,369 @@
...
+`--constraint` (Alias `-c`) Constraints for provider auction in the form of "label=value". If host-id is supplied, this list is ignored
+
+`--auction-timeout-ms` Timeout to await an auction response, defaults to 2000 milliseconds [default: 2000]
+
+`--config-json` Path to provider configuration JSON file
+
+`--skip-wait` By default, the command will wait until the provider has been started. If this flag is passed, the command will return immediately after acknowledgement from the host, without waiting for the provider to start. If this flag is omitted, the timeout will be adjusted to 30 seconds to account for provider download times
+
+## `stop`
+Stop an actor provider or a host. Following are the available subcommands:
```
Comment (thomastaylor312) [inline docs/cli/ctl.md:227]: Nit: Add a newline above this

File: docs/cli/ctl.md [hunk 15]
```diff
@@ -0,0 +1,369 @@
...
+
+`--js-domain` JS domain for wasmcloud control interface. Defaults to None [env: WASMCLOUD_JS_DOMAIN]
+
+`--lattice-prefix` (Alias `-x`) Lattice prefix for wasmcloud control interface, defaults to "default" [env: WASMCLOUD_LATTICE_PREFIX=]
+
+`--timeout-ms` (Alias `-ot`) Timeout length to await a control interface response, defaults to 2000 milliseconds [env: WASMCLOUD_CTL_TIMEOUT_MS=] [default: 2000]
+
+`--context` Path to a context with values to use for CTL connection and authentication
+
+`--skip-wait` By default, the command will wait until the actor/provider has been started. If this flag is passed, the command will return immediately after acknowledgement from the host, without waiting for the provider to start. If this flag is omitted, the timeout will be adjusted to 30 seconds to account for provider download times and  5 seconds to account for actor download times.
```
Comment (thomastaylor312) [inline docs/cli/ctl.md]: Remove extra space after "provider download times and"

File: docs/cli/ctl.md [hunk 16]
```diff
@@ -0,0 +1,369 @@
...
+
+`--ctl-seed` Seed file or literal for CTL authentication. Must be supplied with ctl_jwt [env: WASMCLOUD_CTL_SEED]
+
+`--ctl-credsfile` Credsfile for CTL authentication. Combines ctl_seed and ctl_jwt. See https://docs.nats.io/developing-with-nats/security/creds for details [env: WASH_CTL_CREDS]
+
+`--js-domain` JS domain for wasmcloud control interface. Defaults to None [env: WASMCLOUD_JS_DOMAIN]
+
+`--lattice-prefix` (Alias `-x`) Lattice prefix for wasmcloud control interface, defaults to "default" [env: WASMCLOUD_LATTICE_PREFIX=]
+
+`--timeout-ms` (Alias `-ot`) Timeout length to await a control interface response, defaults to 2000 milliseconds [env: WASMCLOUD_CTL_TIMEOUT_MS=] [default: 2000]
```
Comment (thomastaylor312) [inline docs/cli/ctl.md]: I think the alias here is just `-t`

File: docs/cli/ctx.md [hunk 17]
```diff
@@ -0,0 +1,84 @@
+---
+title: "wash ctx"
+draft: false
+sidebar_position: 7
+description: "wash ctx command reference"
+--- 
+
+`wash ctx` provides a user with tools to interact with wasmCloud host configuration contexts. By default, the host configuration context is stored in a `host_config.json` file in your `$HOME/.wash/contexts` directory. This command will help you to interactively manage host contexts by performing operations such as addition, removal and editting of contexts. Following are the available subcommands unser `wash ctx`:
```
Comment (thomastaylor312) [inline docs/cli/ctx.md]: s/editting/editing

s/unser/under

File: docs/cli/ctx.md [hunk 18]
```diff
@@ -0,0 +1,84 @@
...
+
+#### Options
+`--directory` Location of context files for managing. Defaults to $WASH_CONTEXTS ($HOME/.wash/contexts) [env: WASH_CONTEXTS]
+
+`--output` (Alias `-o`) Specify output format (text or json) [default: text]
+
+`--interactive` (Alias `-i`) Create the context in an interactive terminal prompt, instead of an autogenerated default context
+
+### `default`
+Set the default context for using in your host operations.
```
Comment (thomastaylor312) [inline docs/cli/ctx.md]: This reads a bit awkwardly to me. Maybe: "Set the default context for use in your lattice operations"

File: docs/cli/dev.md [hunk 19]
```diff
@@ -0,0 +1,114 @@
...
+draft: false
+sidebar_position: 8
+description: "wash dev command reference"
+--- 
+
+Starts a local dev loop for an actor. Takes a local `wasmcloud.toml` file to deploy the sepcified actor and accepts a host ID to specify the host to deploy on. This is an experimental feature and needs `--experimental` flag to run.
+
+### Usage
+```
+wash dev --experimental --host-id=<you-host-id>
```
Comment (thomastaylor312) [inline docs/cli/dev.md]: s/you/your

File: docs/cli/down.md [hunk 20]
```diff
@@ -0,0 +1,41 @@
+---
+title: "wash down"
+draft: false
+sidebar_position: 9
+description: "wash down command reference"
+--- 
+
+This command will stop all the resources started by `wash up`. Primarily it will stop the wasmCloud host and NATS leaf that were started on your local machine. If there is a single host running on your machine, `wash down` will stop that singular host. In case of multiple hosts, the user gets a choice to stop a specific host or all the running hosts on the system. To stop a specific host in a multiple host scenario, pass the `--host-id` flag to the command. To stop currently running hosts, pass the `--all` flag.
```
Comment (thomastaylor312) [inline docs/cli/down.md]: Nit: Add comma after "Primarily"

File: docs/cli/drain.md [hunk 21]
```diff
@@ -0,0 +1,54 @@
+---
+title: "wash drain"
+draft: false
+sidebar_position: 10
+description: "wash drain command reference"
+--- 
+
+Wash drain helps you in managing contents of local wasmCloud caches. This command mainly allows you to clear caches selectively for various items downloaded while working on a wasmCloud environment. Following are the subcommands available under `wash drain`:
```
Comment (thomastaylor312) [inline docs/cli/drain.md]: Nit: remove "mainly" for cleaner reading

File: docs/cli/get.md [hunk 22]
```diff
@@ -0,0 +1,79 @@
...
+This subcommand will retrieve all the known links in the lattice. It will give you information about them such as the link name, the associated actor and provider ID and its contract ID.
+
+#### Usage
+```
+wash get links
+```
+
+### `hosts`
+
+This subcommand retrieves all the hosts currently running in the lattice. You will be able to view each hosts host ID and its up time.
```
Comment (thomastaylor312) [inline docs/cli/get.md]: Should be "uptime" I believe. If spell check is telling you no, then don't worry about it

File: docs/cli/get.md [hunk 23]
```diff
@@ -0,0 +1,79 @@
...
+
+`--ctl-seed` Seed file or literal for CTL authentication. Must be supplied with ctl_jwt [env: WASMCLOUD_CTL_SEED]
+
+`--ctl-credsfile` Credsfile for CTL authentication. Combines ctl_seed and ctl_jwt. See https://docs.nats.io/developing-with-nats/security/creds for details [env: WASH_CTL_CREDS]
+
+`--js-domain` JS domain for wasmcloud control interface. Defaults to None [env: WASMCLOUD_JS_DOMAIN]
+
+`--lattice-prefix` (Alias `-x`) Lattice prefix for wasmcloud control interface, defaults to "default" [env: WASMCLOUD_LATTICE_PREFIX=]
+
+`--timeout-ms` (Alias `-ot`) Timeout length to await a control interface response, defaults to 2000 milliseconds [env: WASMCLOUD_CTL_TIMEOUT_MS=] [default: 2000]
```
Comment (thomastaylor312) [inline docs/cli/get.md]: Should be just `-t`

File: docs/cli/keys.md [hunk 24]
```diff
@@ -0,0 +1,52 @@
...
+--- 
+
+This command helps a user to generate NATS keys and manage them. Following are the subcommands available under `wash keys`.
+
+- `gen`
+- `get`
+- `list`
+
+### `gen`
+`gen`, short for generate. will generate a secret and public key pair. The type of key pair needs to be specified. The following ke types can be used:
```
Comment (thomastaylor312) [inline docs/cli/keys.md:16]: Nit: Newline before this line

File: docs/cli/keys.md [hunk 25]
```diff
@@ -0,0 +1,52 @@
...
+- cluster
+
+#### Usage
+```
+wash keys gen user
+wash keys gen server
+```
+
+### `get`
+This subcommand retrives a key pair and prints its contents.
```
Comment (thomastaylor312) [inline docs/cli/keys.md]: s/retrives/retrieves

File: docs/cli/link.md [hunk 26]
```diff
@@ -0,0 +1,61 @@
...
+
+`--ctl-seed` Seed file or literal for CTL authentication. Must be supplied with ctl_jwt [env: WASMCLOUD_CTL_SEED]
+
+`--ctl-credsfile` Credsfile for CTL authentication. Combines ctl_seed and ctl_jwt. See https://docs.nats.io/developing-with-nats/security/creds for details [env: WASH_CTL_CREDS]
+
+`--js-domain` JS domain for wasmcloud control interface. Defaults to None [env: WASMCLOUD_JS_DOMAIN]
+
+`--lattice-prefix` (Alias `-x`) Lattice prefix for wasmcloud control interface, defaults to "default" [env: WASMCLOUD_LATTICE_PREFIX=]
+
+`--timeout-ms` (Alias `-ot`) Timeout length to await a control interface response, defaults to 2000 milliseconds [env: WASMCLOUD_CTL_TIMEOUT_MS=] [default: 2000]
```
Comment (thomastaylor312) [inline docs/cli/link.md]: Should be `-t`. Probably should just do a find and replace all on `-ot`

File: docs/cli/reg.md [hunk 27]
```diff
@@ -0,0 +1,92 @@
+---
+title: "wash reg"
+draft: false
+sidebar_position: 18
+description: "wash reg command reference"
+--- 
+
+`wash reg` allows users to interact with OCI compliant registries and their artifacts. A user may pull artifacts from the registry to use in their projects. They may also push their own projects to registries to use in various constellations. Following are the subcommands available under `wash reg`.
```
Comment (thomastaylor312) [inline docs/cli/reg.md]: s/constellations/lattices

File: docs/cli/scale.md [hunk 28]
```diff
@@ -0,0 +1,41 @@
...
+sidebar_position: 22
+description: "wash scale command reference"
+--- 
+
+`wash scale` supports scaling the actors in your lattice to handle concurrent requests. It accepts the host ID on which the actor is running and the reference of the actor to scale as required arguments. It also accepts a `--max-concurrent` value that specifies the maximum number of instances this actor can run concurrently. By default, the value is an unbounded level of concurrency. A value of zero is equivalent to stopping the actor.
+
+### Usage
+```
+wash scale actor <your-host-id> wasmcloud.azurecr.io/echo:0.3.7
+wash scale actor <your-host-id> wasmcloud.azurecr.io/echo:0.3.7 --max-concurrent=20
```
Comment (thomastaylor312) [inline docs/cli/scale.md:13]: Consider including an example that shows stopping the actor by specifying `0`

File: docs/cli/spy.md [hunk 29]
```diff
@@ -0,0 +1,37 @@
+---
+title: "wash spy"
+draft: false
+sidebar_position: 21
+description: "wash spy command reference"
+--- 
+
+This command will spy on all invocation between an actor and its linked providers. This is an experimental command and needs the `--experimental` flag passed to it. The user will pass the actor ID or actor name to this command to spy on. If an actor name is passed, it will be resolved to an ID. If multiple actors have the same name, the user will be prompted to pick the desired one.
```
Comment (thomastaylor312) [inline docs/cli/spy.md]: s/invocation/invocations

File: docs/cli/start.md [hunk 30]
```diff
@@ -0,0 +1,52 @@
+---
+title: "wash start"
+draft: false
+sidebar_position: 23
+description: "wash start command reference"
+--- 
+
+`wash start` assists in launching an actor or a provider from an OCI reference on a host. The user may specify if they want to start an actor or a provider and pass the OCI reference of that entity to the command. By default, the actor or provider are auctioned in the lattice for a suitable host. To specify a host to launch the actor or provider, pass the host ID using the `--host-id` flag. Following are the available subcommads:
```
Comment (thomastaylor312) [inline docs/cli/start.md]: We should also document the ability to use a file with the `file://` prefix as well as the requirement for starting the host with the right environment variables to allow it

File: docs/cli/stop.md [hunk 31]
```diff
@@ -0,0 +1,65 @@
+---
+title: "wash stop"
+draft: false
+sidebar_position: 24
+description: "wash stop command reference"
+--- 
+
+`wash stop` assists in stopping an actor or a provider from an OCI reference on a host. `wash stop` also helps in stopping hosts. The user may specify if they want to stop an actor or a provider and pass the OCI reference of that entity to the command. To specify a host to launch the actor or provider, pass the host ID using the `--host-id` flag. Following are the available subcommads:
```
Comment (thomastaylor312) [inline docs/cli/stop.md]: Consider combining the first two sentences: "`wash stop` allows you to stop actors, providers, and hosts"

Also: s/launch/stop

File: docs/cli/up.md [hunk 32]
```diff
@@ -0,0 +1,161 @@
+---
+title: "wash up"
+draft: false
+sidebar_position: 26
+description: "wash up command reference"
+---
+
+To get started with wasmCloud, we're going to start off with spinning up a NATS Server and along with it a wasmCloud host. This is done by running the `wash up` command. It starts a NATS server (if you don't have one running) on port `4222` and a wasmcloud host. If you successively run `wash up` it will spawn multiple wasmCloud hosts that will connect to the originally created NATS server. By default, the host runs in an interactive mode in your terminal. If you add a `-d` flag to this command, the host will run in a detached mode.
```
Comment (thomastaylor312) [inline docs/cli/up.md]: First 2 sentences feel more like the intro to a tutorial rather than docs. Maybe reword to something like "`wash up` lets you easily startup a wasmCloud host to play around or develop with.

A couple other things:
- There is a wasmcloud that should be wasmCloud
- Might be good to mention that it spins up wadm as well

**Review by thomastaylor312 — CHANGES_REQUESTED** [review]
Content is mostly all good to go, just some spelling and rewording fixes for the most part

File: docs/cli/claims.md (hunk 10 above)
Comment (aish-where-ya) [reply docs/cli/claims.md:217]: Afaik the only global flags are `--output`, `--help` and `--experimental`. We already make a note about `--help` on the main `index.md` page for wash. But the other two flags cannot be used without subcommands so they don't exactly fit in there.

File: docs/cli/ctl.md (hunk 13 above)
Comment (aish-where-ya) [reply docs/cli/ctl.md:52]: Yea. Since these options were repeated for all the subcommands of `wash ctl get`, I put them together

File: docs/cli/capture.md [hunk 33]
```diff
@@ -0,0 +1,59 @@
+---
+title: "wash capture"
+draft: false
+sidebar_position: 3
+description: "wash capture command reference"
+--- 
+
+Wash capture helps in capturing snapshots of the state of a lattice and assists in debugging cluster invocations using operations like replay. This is currently an experimental feature and you would have to pass the `--experimental` flag or set the `WASH_EXPERIMENTAL` environment variable to true. This tool provides three options - enable capture mode, disable capture mode or replay a captured file. A user must first enable capture mode by passing the `--enable` flag. Once capture mode is enabled, a user can capture snapshots at any time by running wash capture --experimental, which will download the snapshot file to the current directory. The snapshot files can be viewed by passing the file path to the `replay` subcommand. To disable capture mode, pass the `--disable` flag.
```
Comment (thomastaylor312) [inline docs/cli/capture.md]: Might want to wrap `wash capture --experimental` in backticks here

File: docs/cli/start.md [hunk 34]
```diff
@@ -0,0 +1,56 @@
+---
+title: "wash start"
+draft: false
+sidebar_position: 23
+description: "wash start command reference"
+--- 
+
+`wash start` assists in launching an actor or a provider from an OCI reference on a host. The user may specify if they want to start an actor or a provider and pass the OCI reference of that entity to the command. By default, the actor or provider are auctioned in the lattice for a suitable host. To specify a host to launch the actor or provider, pass the host ID using the `--host-id` flag. Actors or providers can also be started from local files using the prefix `file://` followed by the file path. By default, loading from files is permitted and can be disabled using the `WASMCLOUD_ALLOW_FILE_LOAD` environment variable. Following are the available subcommads:
```
Comment (thomastaylor312) [inline docs/cli/start.md]: One more tiny change: The default is actually to not allow file loading on the host. However, for `wash up` (since it is mostly using for development), it is the default. It would be good to clarify here

**Review by thomastaylor312 — APPROVED** [review]
